test/_globals-test.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const supertest = require('supertest');
+const fs = require("fs");
+
+const config = require('wild-config');
+
+const server = supertest.agent(`http://127.0.0.1:${config.api.port}`);
+
+const titles = [];
+const unsupportedTitles = [];
+
+// global beforeEach to run before EVERY test
+beforeEach("Get test data before each test", async function () {
+    const currentTestTitle = this.test.ctx.currentTest.title; // eslint-disable-line no-invalid-this
+    if (/POST|PUT|DELETE|GET|post|put|delete|get/.test(currentTestTitle) && /success|failure/.test(currentTestTitle)) {
+        titles.push(currentTestTitle);
+    } else {
+        unsupportedTitles.push(currentTestTitle);
+    }
+});
+
+// eslint-disable-next-line no-undef
+after("Generate test overview table after all tests", async () => {
+    try {
+        const data = await server.get("/api-methods/args");
+        
+        const routes = data.body;
+
+        const mapApiMethodToSpec = {};
+        let content = "| API path | API method | Test count | Has positive test? | Has Negative test? |\n";
+        content += "| --- | :---: | --- | --- | --- | \n"
+
+        for (const routeName in routes) {
+            const route = routes[routeName];
+            const method = route.spec.method;
+            const path = route.spec.path;
+
+            mapApiMethodToSpec[`${method.toLowerCase()}_${path}`] = {method, path, name: route.spec.name || route.name, testCount: 0, positiveTestCount: 0, negativeTestCount: 0};
+        }
+
+        for (const title of titles) {
+            const titleSplit = title.split(" ");
```

Comment:
It is safer to use a regex for splitting.
```
title.split(/\s+/);
```

Reply:
done